modules/installation-azure-permissions.adoc

```diff
@@ -6,10 +6,10 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-Your Microsoft Azure account must have the following roles for the subscription
-that you use:
+Your Microsoft Azure, which will be used to create the service principal used by the installer and cluster, must have the following roles for the subscription that you use:
```

Comment:
I was a bit confused with this. Breaking up the sentence may help make it clearer:

"{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you can create a service principal, your Azure account subscription must have the following roles:"

Reply:
I agree. Above sentence is clearer

Reply:
Addressed